sycl/source/detail/queue_impl.cpp

```diff
@@ -278,6 +278,14 @@ void queue_impl::wait(const detail::code_location &CodeLoc) {
   TelemetryEvent = instrumentationProlog(CodeLoc, Name, StreamID, IId);
 #endif
 
+#if 1
```

Comment:
Think we should remove this, no other part of this PR is guarded by the preprocessor